BlueprintUICommonControls/Sources/PointerInteraction.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  PointerInteraction.swift
+//  BlueprintUICommonControls
```

Comment:
Thoughts on in here vs in BlueprintUI? I think BlueprintUI might actually make more sense since this isn't really a control; it's a general part of UIKit you can apply to any view.

Reply:
Agree, will do

Reply:
Feels more like a CommonControls thing to me, for the same reason that's where Tappable lives. It's not part of the core layout stuff.

Reply:
In my brain, CommonControls is more of "these are the vanilla UIKit controls" that are basically the Blueprint version of UIKit. But things that on their own dont have a visual style like button, imo, should live in just core Blueprint. Eg, I'd move things like `ScrollView` into core Blueprint.